### PR TITLE
Images can eagerly load in IDE mode

### PR DESCRIPTION
**Pull request by hqhhuang**
Bug/issue #, if applicable: rdar://107320467

## Summary
Images should be allowed to eagerly load by default in IDE mode.

### CODE REVIEW

File: src/stores/AppStore.js [hunk 1]
```diff
@@ -22,7 +22,8 @@ const defaultColorScheme = supportsAutoColorScheme ? ColorScheme.auto : ColorSch
 
 export default {
   state: {
-    imageLoadingStrategy: ImageLoadingStrategy.lazy,
+    imageLoadingStrategy: process.env.VUE_APP_TARGET === 'ide'
```
Comment (mportiz08) [inline src/stores/AppStore.js:25]: Can you add a test for this?

**Review by mportiz08 — APPROVED** [review]

**Comment by mportiz08** [comment]
@swift-ci test

File: src/stores/AppStore.js (hunk 1 above)
Comment (dobromir-hristov) [reply src/stores/AppStore.js:25]: If you want to clean it up a bit you can extract the load strategy to a variable:

```js
const strategy = process.env.VUE_APP_TARGET === 'ide'
      ? ImageLoadingStrategy.eager : ImageLoadingStrategy.lazy
```

**Review by dobromir-hristov — APPROVED** [review]
Just left a minor comment, but overall I think its good.

If we switch to Pinia at some point, lots of things might get simpler with these stores, like resetting for example.

File: src/stores/AppStore.js (hunk 1 above)
Comment (hqhhuang) [reply src/stores/AppStore.js:25]: Discussed with Dobri - we need to make sure to re-import the file during testing, which isn't worth it to me. So I'm gonna keep it this way :)

**Comment by hqhhuang** [comment]
@swift-ci test

**Pull request merged**